.gitignore

```diff
@@ -219,3 +219,58 @@ config.h
 callgrind.out.*
 data/dist
 clean.sh
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
+/build/
```

Comment:
What on earth is this?

Reply:
See 424a0cff3a65a201e5f7c449b43c0df0ae08274e. Should be cleaned up.